.eslintrc

```diff
@@ -78,7 +81,11 @@
 		"semi": [1, "always"], // require or disallow use of semicolons instead of ASI
 		"semi-spacing": [2, {"before": false, "after": true}],
 		"space-infix-ops": 1, // require spaces around operators
-		"space-return-throw-case": 1, // require a space after return, throw, and case
+		"keyword-spacing": [2, { "overrides": {
+  "if": { "after": false },
+  "for": { "after": false },
+  "while": { "after": false }
+} }], // require a space after return, throw, and case
```

Comment:
Looks like the indentation for the `keyword-spacing` rule declaration got a bit messed up.